### PR TITLE
Zina

### PR DESCRIPTION
**Pull request by untuzina**
please review

### CODE REVIEW

File: src/Practice01.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+public class Practice01 {
+
+    public static void main(String[] args) {
+
+        System.out.println("Hello Cydeo friends!");
```
Comment (stars2026) [inline src/Practice01.java:5]: Good job. Keep up the good work.

File: src/Practice01.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+    }
```
Comment (stars2026) [inline src/Practice01.java:8]: Good. Thank you

**Review by Ihor0777 — APPROVED** [review]

**Review by stars2026 — APPROVED** [review]

**Review by stars2026 — COMMENTED** [review]
Good to go !

**Review by stars2026 — COMMENTED** [review]
No correction was made

File: src/Practice01.java (hunk 1 above)
Comment (stars2026) [reply src/Practice01.java:5]: Thanks

**Pull request merged**